You can try using the `tz_localize` method to set the timezone to UTC and then use `tz_convert` to convert it to a timezone-unaware datetime object. Here's an example:

```python
import pandas as pd

def removeTimezone(df):
    df['time'] = df['time'].dt.tz_localize(None).dt.tz_convert(None)
    return df
```

This function takes a DataFrame `df` as input and removes the timezone information from the "time" column. It first uses `tz_localize` to set the timezone to UTC, and then uses `tz_convert` to convert it to a timezone-unaware datetime object. Finally, it returns the modified DataFrame.

You can call this function before saving the DataFrame to Excel to avoid the `ValueError` you're encountering.